tests/unit/helpers.js

```diff
@@ -378,11 +400,13 @@ export const createSecondaryHeroShelf = ({
   };
 };
 
-export const createHeroShelves = ({
+export const createHomeShelves = ({
+  resultsProps = [],
   primaryProps = {},
   secondaryProps = {},
 } = {}) => {
   return {
+    results: createResultsShelf(resultsProps),
```

Comment:
```suggestion
    results: results || createResultsShelf(),
```

Reply:
Applying these three suggestions to this file will fix those errors. It changes the interface for `createHomeShelves`, but I think that's fine.